lib/locales/es.ts

```diff
@@ -286,18 +267,22 @@ export const locale: Locale = {
     hedwig_of_silesia_religious: 'Santa Eduviges, Religiosa',
     helena_of_constantinople: 'Santa Elena',
     holy_child_of_cebu: 'Divino Niño', // TODO: Should it not be rather `Santo Niño de Cebú`?
+    holy_family: 'Sagrada Familia',
     holy_innocents_martyrs: 'Santos Inocentes, Mártires',
+    holy_saturday: 'Sábado Santo',
+    holy_thursday: 'Jueves Santo',
     ignatius_of_antioch_bishop: 'San Ignacio de Antioquía, Obispo y Mártir',
     ignatius_of_loyola_priest: 'San Ignacio de Loyola, Sacerdote',
+    immaculate_conception_of_mary: 'Inmaculada Concepción',
```

Comment:
This should still refer to Mary as `Inmaculada Concepción de María`

Reply:
Thanks!